frontend/src/styles/styles.scss

```diff
@@ -76,6 +76,7 @@
   height: 100%;
   background-size: cover;
   background-position: center;
+  height: 100%;
```

Comment:
do we need this here twice? its on line 76 as well

Reply:
u right